### ISSUE TITLE
Time and Temp Display

### ISSUE THREAD

**Issue by cucanunes** [Lunatixz/KODI_Addons#4]
Hi, for those using plasma, Time and Temp being on screen all the time may cause screen burn issues.
What about add a option to fade it together with Google Earth logo and location?

**Comment by Lunatixz** [comment]
I believe the logo is transparent? So even though it looks "white" on closer inspection your see the pixels shifting to keep up with the background.

**Comment by cucanunes** [comment]
Sorry, I mean:
*animation* fade the time and temp, together with the Google logo and location, 
Google logo and location goes away 15000milisecs after show on screen, and then when a new image is presented on screen, both are presented together for another 15000milisecs and go away....

This is the line I'm talking about on default.xml of 1080i skin folder:
<animation effect="fade" start="100" end="0" time="15000" condition="True">Conditional</animation>

**Comment by cucanunes** [comment]
`<animation effect="fade" start="100" end="0" time="15000" condition="True">Conditional</animation>`

**Comment by Lunatixz** [comment]
add the aforementioned animation effect under lines 85, 98 =)

**Comment by cucanunes** [comment]
Uh! I tried many combinations, including adding the `<visible>` condition inside the animation condition, but I have very limited knowledge in Kodi environment to make it work.
The best I have made was, remove the  `<visible>` tag, and added same conditions on  `<animation>` tag. Works, but only for the first google earth image, after the second, time and temp don't appear again, just Google Logo and location... :-(
